Bug and Feature Request Templates + MLH Theme + Readme (#9068)
ohmyzsh/ohmyzsh: README.md

### Prerequisites

* A Unix-like operating system: macOS, Linux, BSD. On Windows: WSL2 is preferred, but cygwin or msys also mostly work.

> * [Zsh](https://www.zsh.org) should be installed (v4.3.9 or more recent is fine but we prefer 5.0.8 and newer). If not pre-installed (run `zsh --version` to confirm), check the following wiki instructions here: [Installing ZSH](https://github.com/ohmyzsh/ohmyzsh/wiki/Installing-ZSH)

Reviewer: Why 5.0.8? Haven't done my research on this that's why I'm asking.

Author: I noticed when I was backtracking the version changes of Z-Shell that they implemented new key bindings. It also improved speed.

Note: I didn't look at the code for any of the z-shell stuff. I just looked changelog by changelog.

* `curl` or `wget` should be installed
* `git` should be installed (recommended v2.4.11 or higher)

### Basic Installation

> Oh My Zsh is installed by running one of the following commands in your terminal. You can install this via the command-line with either `curl` or `wget` whatever one that is installed on your machine.

Reviewer: ```suggestion
Oh My Zsh is installed by running one of the following commands in your terminal. You can install this via the command-line with either `curl` or `wget`, whichever is installed on your machine.
```

)
```

> _Note that the plugins are separated by whitespace (spacebar/enterkey). **Do not** use commas between them or it will break._

Reviewer: ```suggestion
_Note that the plugins are separated by whitespace (spaces, tabs, new lines...). **Do not** use commas between them or it will break._
```

#### Using Plugins

> Most plugins (should! we're working on this) include a __README__, which documents how to use them. This README should show the aliases and extra goodies that are inclued in that particular plugin.

Reviewer: Some plugins don't add aliases, can this be more generic? Also we can get rid of the `should! we're working on this` part. It is a statement: each plugin has its own README documenting it, we finally got to having one for each.

Author: Fixed, please check and make sure that I worded it properly.

## Advanced Topics

> If you're the type that likes to get their hands dirty (and break stuff), these sections might resonate.

Reviewer: No reason to get negative, sounds a bit judgy.

Author: Fixed

#### Custom Directory

> The default location is `~/.oh-my-zsh` (hidden in your home directory but can access by `cd ~/.oh-my-zsh`)

Reviewer: ```suggestion
The default location is `~/.oh-my-zsh` (hidden in your home directory, you can access it with `cd ~/.oh-my-zsh`)
```
